WordPress/gutenberg
# Site Editor: Fix template parts 'Reset' action

## What?
Fixes #62604.

PR fixes the "is eligible" condition for template parts and makes the "Reset" action work properly.

## Why?
The action was no longer available for template part entities.

## Testing Instructions
1. Open the Site Editor.
2. Make changes to the template parts provided by the theme.
3. Confirm you can reset template parts when editing one.
4. Confirm you can reset one or more template parts in data views.
5. Confirm that the reset action isn't available for parts created by the users.

### Testing Instructions for Keyboard
Same.

## Screenshots or screencast 
**Editor sidebar**
![CleanShot 2024-06-28 at 09 19 35](https://github.com/WordPress/gutenberg/assets/240569/850e73cb-cdc5-4d9a-b666-306499594084)

**DataViews**
![CleanShot 2024-06-28 at 09 19 18](https://github.com/WordPress/gutenberg/assets/240569/87ee17f2-855d-4efd-974b-c8acadbde05a)
